manifests/a/A4TECH/Bloody7/V2022.0805/A4TECH.Bloody7.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: A4TECH.Bloody7
+PackageVersion: V2022.0805
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://www.a4tech.com.tw/download/BloodyMouse/Bloody7_V2022.0805_MUI.exe
+  InstallerSha256: 7D02D9F654A9B60FAE5E7C22D493D8EDA4711AA5E4CC4296A6583AD544334550
+  InstallerSwitches:
+    Silent: /qn

```

Comment:
Please also include the `SilentWithProgress` switch

![image](https://user-images.githubusercontent.com/12611259/204305477-4ea35104-294f-40ed-bc79-212a15726a77.png)

Reply:
> Please also include the `SilentWithProgress` switch
> 
What flag should I add to include the "SilentWithProgress" switch?

Reply:
The `SilentWithProgress` switch might be the same as the `Silent` switch.

I believe the issue Trenly is seeing is actually caused by the installer being launched by a self-extracting Winrar archive. The `Silent` switch /S hides the unpacking operation, but the installer is still in attended mode. After copying the installer files to another location and querying it, I couldn't find any switches to enable unattended mode, nor even a dialog of available switches. Unfortunately, packages must have a fully unattended install mode to be available through Winget.